dashboard/test/ui/support/dashboard_helpers.rb

```diff
@@ -1,7 +1,11 @@
 module DashboardHelpers
   # Requires the full rails environment. Use sparingly, known to take 20-30s.
   def require_rails_env
+    puts 'Requiring rails env'
+    start = Time.now
     require File.expand_path('../../../../config/environment.rb', __FILE__)
+    finish = Time.now
+    puts "Requiring rails env took #{finish - start} seconds"
```

Comment:
Actually - puts is the right answer here. In runner.rb, stdout is the logger that gets uploaded to s3, and puts writes to stdout. This is the convention used by other parts as well (see debug cookies in steps.rb)